client/oidc/src/main/java/io/quarkiverse/openapi/generator/oidc/providers/OAuth2AuthenticationProvider.java

```diff
@@ -32,9 +33,12 @@ public OAuth2AuthenticationProvider(String name,
     @Override
     public void filter(ClientRequestContext requestContext) throws IOException {
         if (isTokenPropagation()) {
-            String bearerToken = getTokenForPropagation(requestContext.getHeaders());
-            bearerToken = sanitizeBearerToken(bearerToken);
-            requestContext.getHeaders().add(HttpHeaders.AUTHORIZATION, OidcConstants.BEARER_SCHEME + " " + bearerToken);
+            String bearerToken = sanitizeBearerToken(getTokenForPropagation(requestContext.getHeaders()));
+            if (!isEmptyOrBlank(bearerToken)) {
+                requestContext.getHeaders().remove(HttpHeaders.AUTHORIZATION);
```

Comment:
Do we really need to call remove here?

Reply:
The purpose is that, if quarkus change again, we just change the code in one class, not in three